.github/workflows/chromatic._template.yml

```diff
@@ -0,0 +1,50 @@
+name: "Publish to chromatic"
+on:
+  workflow_call:
+    inputs:
+      working-directory:
+        required: true
+        description: The directory where Storybook should be published from
+        type: string
+    secrets:
+      CHROMATIC_PROJECT_TOKEN:
+        required: true
+        description: Chromatic project token for publishing
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref
+    }}
+  cancel-in-progress: true
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+        # "prepare" hook on `bun install` runs `bun run build`
+      - name: Setup environment
+        uses: ./.github/actions/setup-env
+
+      - name: Code quality checks
+        run: bun run check
+
+      - name: Test
+        run: bun run test
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+        # "prepare" hook on `bun install` runs `bun run build`
+      - name: Setup environment
+        uses: ./.github/actions/setup-env
+
+      - name: Publish to Chromatic
+        uses: chromaui/action@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
```

Comment:
Something is going wrong with passing this token from the react core workflow to the template, see this [error](https://github.com/canonical/ds25/actions/runs/13950596087)

```

Invalid workflow file: .github/workflows/chromatic.ds-react-core.yml#L7
The workflow is not valid. .github/workflows/chromatic.ds-react-core.yml (Line: 7, Col: 11): Secret CHROMATIC_PROJECT_TOKEN is required, but not provided while calling. .github/workflows/chromatic.ds-react-core.yml (Line: 9, Col: 22): Invalid secret, PROJECT_TOKEN is not defined in the referenced workflow.
```

I haven't tested the other two packages, but they may have the same problem

Reply:
Fixed by @advl